.gitattributes

```diff
@@ -0,0 +1,17 @@
+# https://php.watch/articles/composer-gitattributes
+
+# Exclude build/test files from archive
+/.github            export-ignore
+/bin                export-ignore
+/docker             export-ignore
+/tests              export-ignore
+/.editorconfig      export-ignore
+/.gitattributes     export-ignore
+/.gitignore         export-ignore
+/ecs.php            export-ignore
+/Makefile           export-ignore
+/phpstan.neon.dist  export-ignore
+/phpunit.xml.dist   export-ignore
+
+# Configure diff output for .php and .phar files.
```

Comment:
You copied also the comment mentioning phar which is irrelevant here, but this is just a little issue. 🙃

Reply:
Fixing this - https://github.com/lmc-eu/twigx-bundle/pull/18 :-)